smurfs/src/components/Form.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { addSmurf } from '../actions/Action';
+
+const Form = (props) => {
+    
+    const [item, setItem] = useState({name: "", age: "", height: ""});
+    const handleChange = event => setItem({
+        ...item,
+        [event.target.name]:event.target.value
+    });
```

Comment:
for clarity sake `item` could be changed to `newSmurf` or something along those lines.